mmv1/products/edgecontainer/Cluster.yaml

```diff
@@ -0,0 +1,515 @@
+    output: true
+  - !ruby/object:Api::Type::String
+    name: "clusterCaCertificate"
+    description: |
+      The PEM-encoded public certificate of the cluster's CA.
+    output: true
+    sensitive: true
+  - !ruby/object:Api::Type::NestedObject
+    name: "maintenancePolicy"
+    description: |
+      Cluster-wide maintenance policy configuration.
+    default_from_api: true
+    properties:
+      - !ruby/object:Api::Type::NestedObject
+        name: "window"
+        description: |
+          Specifies the maintenance window in which maintenance may be performed.
+        default_from_api: true
+        properties:
+          - !ruby/object:Api::Type::NestedObject
+            name: "recurringWindow"
+            description: |
+              Represents an arbitrary window of time that recurs.
+            default_from_api: true
+            properties:
+              - !ruby/object:Api::Type::NestedObject
+                name: "window"
+                description: |
+                  Represents an arbitrary window of time.
+                default_from_api: true
+                properties:
+                  - !ruby/object:Api::Type::Time
+                    name: "startTime"
+                    description: |
+                      The time that the window first starts.
+                    default_from_api: true
+                  - !ruby/object:Api::Type::Time
+                    name: "endTime"
+                    description: |
+                      The time that the window ends. The end time must take place after the
+                      start time.
+                    default_from_api: true
+              - !ruby/object:Api::Type::String
+                name: "recurrence"
+                description: |
+                  An RRULE (https://tools.ietf.org/html/rfc5545#section-3.8.5.3) for how
+                  this window recurs. They go on for the span of time between the start and
+                  end time.
+                default_from_api: true
+  - !ruby/object:Api::Type::String
+    name: "controlPlaneVersion"
+    description: |
+      The control plane release version.
+    output: true
+  - !ruby/object:Api::Type::String
+    name: "nodeVersion"
+    description: |
+      The lowest release version among all worker nodes. This field can be empty
+      if the cluster does not have any worker nodes.
+    output: true
+  - !ruby/object:Api::Type::NestedObject
+    name: "controlPlane"
+    description: |
+      The configuration of the cluster control plane.
+    properties:
+      - !ruby/object:Api::Type::NestedObject
+        name: "remote"
+        immutable: true
+        exactly_one_of:
+          - control_plane.0.remote
+          - control_plane.0.local
+        description: |
+          Remote control plane configuration.
+        default_from_api: true
+        properties:
+          - !ruby/object:Api::Type::String
+            name: "nodeLocation"
```

Comment:
Here and under `local`, I think you need `default_from_api: true` if it can be populated by the server.

Reply:
Done.